boards/mbed_lpc1768/include/board.h

```diff
@@ -41,24 +41,18 @@ extern "C" {
 #define LED2_PIN            GPIO_PIN(1, 21)
 #define LED3_PIN            GPIO_PIN(1, 23)
 
-#define LED_PORT            (LPC_GPIO1)
-#define LED0_MASK           (BIT18)
-#define LED1_MASK           (BIT20)
-#define LED2_MASK           (BIT21)
-#define LED3_MASK           (BIT23)
-
-#define LED0_ON             (LED_PORT->FIOSET  = LED0_MASK)
-#define LED0_OFF            (LED_PORT->FIOCLR  = LED0_MASK)
-#define LED0_TOGGLE         (LED_PORT->FIOPIN ^= LED0_MASK)
-#define LED1_ON             (LED_PORT->FIOSET  = LED1_MASK)
-#define LED1_OFF            (LED_PORT->FIOCLR  = LED1_MASK)
-#define LED1_TOGGLE         (LED_PORT->FIOPIN ^= LED1_MASK)
-#define LED2_ON             (LED_PORT->FIOSET  = LED2_MASK)
-#define LED2_OFF            (LED_PORT->FIOCLR  = LED2_MASK)
-#define LED2_TOGGLE         (LED_PORT->FIOPIN ^= LED2_MASK)
-#define LED3_ON             (LED_PORT->FIOSET  = LED3_MASK)
-#define LED3_OFF            (LED_PORT->FIOCLR  = LED3_MASK)
-#define LED3_TOGGLE         (LED_PORT->FIOPIN ^= LED3_MASK)
+#define LED0_ON             gpio_clear(LED0_PIN)
```

Comment:
Looks like this is inverted.

Reply:
Fixed.

On the Seeeduino Arch-Pro, they are inverted :-)